pkg/ipfailover/keepalived/generator.go

```diff
@@ -36,7 +36,7 @@ func getClientConfig(path string) (*kclient.Config, error) {
 	}
 
 	if err := kclient.LoadTLSFiles(config); err != nil {
-		fmt.Errorf("Unable to load certificate info using credentials from %q: %v", path, err)
+		return nil, fmt.Errorf("Unable to load certificate info using credentials from %q: %v", path, err)
```

Comment:
@stevekuznetsov  - good catch ... your change is good as the error will percolate up and be handled by ```cmdutil.CheckErr```.